src/tools/r2rdump/UnwindInfo.cs

```diff
@@ -6,11 +6,47 @@
 
 namespace R2RDump
 {
+    public enum REGISTERS
```

Comment:
We should use PascalCasing for managed classes / enums. I know we have some of these already in the dumper, but it will cut down refactoring later.